scenarios/microsoft-azure-arcbox-v1/README.md

```diff
@@ -1,11 +1,10 @@
-# JumpStart ArcBox
+# Jumpstart ArcBox
```

Comment:
these changes should be made to the arcbox-overview.md  (as called out in the configuration.metadata.json) . the readme.md doesn't serve anything. either change the config to pull from the readme or update the arcbox-overview.md file..